### PR TITLE
style: add braces around if/while statements

### PR DESCRIPTION
**Pull request by nashif**
Per guidelines, all statements should have braces around them. We do not
have a CI check for this, so a few went in unnoticed.

Note: used clang-tidy to discover and fix those...

### CODE REVIEW

**Review by tbursztyka — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

File: drivers/wifi/eswifi/eswifi_offload.c [hunk 1]
```diff
@@ -452,8 +452,9 @@ static int eswifi_off_recv(struct net_context *context,
 	k_sem_reset(&socket->read_sem);
 	eswifi_unlock(eswifi);
 
-	if (timeout == K_NO_WAIT)
+	if (timeout == K_NO_WAIT) {
 		return 0;
+}
```
Comment (anangl) [inline drivers/wifi/eswifi/eswifi_offload.c]: Imperfect indentation.

File: tests/drivers/uart/uart_basic_api/src/test_uart_poll.c [hunk 2]
```diff
@@ -22,8 +22,9 @@ static int test_poll_in(void)
 
 	/* Verify uart_poll_in() */
 	while (1) {
-		while (uart_poll_in(uart_dev, &recv_char) < 0)
+		while (uart_poll_in(uart_dev, &recv_char) < 0) {
 			;
+	}
```
Comment (anangl) [inline tests/drivers/uart/uart_basic_api/src/test_uart_poll.c]: This should be also indented one tab more.

**Review by anangl — CHANGES_REQUESTED** [review]

File: drivers/wifi/eswifi/eswifi_offload.c (hunk 1 above)
Comment (dcpleung) [inline drivers/wifi/eswifi/eswifi_offload.c]: missed a tab here.

File: tests/drivers/uart/uart_basic_api/src/test_uart_poll.c (hunk 2 above)
Comment (dcpleung) [inline tests/drivers/uart/uart_basic_api/src/test_uart_poll.c]: missed a tab here.

File: tests/kernel/common/src/clock.c [hunk 3]
```diff
@@ -40,20 +40,22 @@ void test_clock_uptime(void)
 
 	/**TESTPOINT: uptime elapse*/
 	t64 = k_uptime_get();
-	while (k_uptime_get() < (t64 + 5))
+	while (k_uptime_get() < (t64 + 5)) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/clock.c]: Should this one be after #endif?

File: tests/kernel/common/src/clock.c [hunk 4]
```diff
@@ -40,20 +40,22 @@ void test_clock_uptime(void)
...
 
 	/**TESTPOINT: uptime elapse lower 32-bit*/
 	t32 = k_uptime_get_32();
-	while (k_uptime_get_32() < (t32 + 5))
+	while (k_uptime_get_32() < (t32 + 5)) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/clock.c]: Should this one be after #endif?

File: tests/kernel/common/src/clock.c [hunk 5]
```diff
@@ -64,21 +66,23 @@ void test_clock_uptime(void)
 	/**TESTPOINT: uptime delta*/
 	d64 = k_uptime_delta(&d64);
 	/* Note: this will stall if the systick period < 5ms */
-	while (k_uptime_delta(&d64) < 5)
+	while (k_uptime_delta(&d64) < 5) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/clock.c]: Should this one be after #endif?

File: tests/kernel/common/src/clock.c [hunk 6]
```diff
@@ -64,21 +66,23 @@ void test_clock_uptime(void)
...
 	/**TESTPOINT: uptime delta lower 32-bit*/
 	k_uptime_delta_32(&d64);
 	/* Note: this will stall if the systick period < 5ms */
-	while (k_uptime_delta_32(&d64) < 5)
+	while (k_uptime_delta_32(&d64) < 5) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/clock.c]: Should this one be after #endif?

File: tests/kernel/common/src/clock.c [hunk 7]
```diff
@@ -101,22 +105,24 @@ void test_clock_cycle(void)
 	c32 = k_cycle_get_32();
 	/*break if cycle counter wrap around*/
 	while (k_cycle_get_32() > c32 &&
-	       k_cycle_get_32() < (c32 + sys_clock_hw_cycles_per_tick()))
+	       k_cycle_get_32() < (c32 + sys_clock_hw_cycles_per_tick())) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/clock.c]: Should this one be after #endif?

File: tests/kernel/common/src/clock.c [hunk 8]
```diff
@@ -101,22 +105,24 @@ void test_clock_cycle(void)
...
 	c0 = k_cycle_get_32();
 	ALIGN_MS_BOUNDARY;
 	t32 = k_uptime_get_32();
-	while (t32 == k_uptime_get_32())
+	while (t32 == k_uptime_get_32()) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/clock.c]: Should this one be after #endif?

File: tests/kernel/common/src/timeout_order.c [hunk 9]
```diff
@@ -65,11 +65,12 @@ void test_timeout_order(void)
 	u32_t uptime = k_uptime_get_32();
 
 	/* sync on tick */
-	while (uptime == k_uptime_get_32())
+	while (uptime == k_uptime_get_32()) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
+	}
```
Comment (dcpleung) [inline tests/kernel/common/src/timeout_order.c]: Should this one be after #endif?

**Review by dcpleung — CHANGES_REQUESTED** [review]

**Review by anangl — APPROVED** [review]

File: tests/kernel/common/src/timeout_order.c (hunk 9 above)
Comment (nashif) [reply tests/kernel/common/src/timeout_order.c]: clang-tidy was not that smart after all :(

**Review by MaureenHelm — APPROVED** [review]

File: drivers/adc/adc_intel_quark_d2000.c [hunk 10]
```diff
@@ -136,8 +136,9 @@ static void adc_quark_d2000_set_mode(struct device *dev, int mode)
 
 	/* Set mode and wait for change */
 	adc_regs->op_mode = mode;
-	while ((adc_regs->op_mode & ADC_OP_MODE_OM_MASK) != mode)
+	while ((adc_regs->op_mode & ADC_OP_MODE_OM_MASK) != mode) {
 		;
```
Comment (carlescufi) [inline drivers/adc/adc_intel_quark_d2000.c]: I don't think you need the `;` anymore

File: drivers/bluetooth/hci/ipm_stm32wb.c [hunk 11]
```diff
@@ -302,8 +302,10 @@ static void start_ble_rf(void)
 
 	/* Select LSE clock */
 	LL_RCC_LSE_Enable();
-	while (!LL_RCC_LSE_IsReady())
+	while (!LL_RCC_LSE_IsReady()) {
 		;
```
Comment (carlescufi) [inline drivers/bluetooth/hci/ipm_stm32wb.c]: I don't think you need the `;` anymore

File: drivers/bluetooth/hci/ipm_stm32wb.c [hunk 12]
```diff
@@ -312,8 +314,10 @@ static void start_ble_rf(void)
 	LL_RCC_LSI1_Disable();
 	/* Set RNG on HSI48 */
 	LL_RCC_HSI48_Enable();
-	while (!LL_RCC_HSI48_IsReady())
+	while (!LL_RCC_HSI48_IsReady()) {
 		;
```
Comment (carlescufi) [inline drivers/bluetooth/hci/ipm_stm32wb.c]: I don't think you need the `;` anymore

File: drivers/adc/adc_intel_quark_d2000.c [hunk 13]
```diff
@@ -149,8 +150,9 @@ static void adc_quark_d2000_set_mode(struct device *dev, int mode)
 
 		/* run dummy conversion and wait for completion */
 		adc_regs->cmd = (ADC_CMD_IE | ADC_CMD_START_SINGLE);
-		while (!(adc_regs->intr_status & ADC_INTR_STATUS_CC))
+		while (!(adc_regs->intr_status & ADC_INTR_STATUS_CC)) {
 			;
```
Comment (carlescufi) [inline drivers/adc/adc_intel_quark_d2000.c]: I don't think you need the `;` anymore

File: drivers/counter/counter_ll_stm32_rtc.c [hunk 14]
```diff
@@ -286,8 +286,9 @@ static int rtc_stm32_init(struct device *dev)
 	LL_RCC_LSE_Enable();
 
 	/* Wait until LSE is ready */
-	while (LL_RCC_LSE_IsReady() != 1)
+	while (LL_RCC_LSE_IsReady() != 1) {
 		;
```
Comment (carlescufi) [inline drivers/counter/counter_ll_stm32_rtc.c]: I don't think you need the `;` anymore

File: drivers/i2c/i2c_sifive.c [hunk 15]
```diff
@@ -80,8 +80,9 @@ static int i2c_sifive_send_addr(struct device *dev,
 	u8_t command = 0U;
 
 	/* Wait for a previous transfer to complete */
-	while (i2c_sifive_busy(dev))
+	while (i2c_sifive_busy(dev)) {
 		;
```
Comment (carlescufi) [inline drivers/i2c/i2c_sifive.c]: I don't think you need the `;` anymore

File: drivers/i2c/i2c_sifive.c [hunk 16]
```diff
@@ -92,8 +93,9 @@ static int i2c_sifive_send_addr(struct device *dev,
 	/* Write the command register to start the transfer */
 	sys_write8(command, I2C_REG(config, REG_COMMAND));
 
-	while (i2c_sifive_busy(dev))
+	while (i2c_sifive_busy(dev)) {
 		;
```
Comment (carlescufi) [inline drivers/i2c/i2c_sifive.c]: I don't think you need the `;` anymore

File: drivers/i2c/i2c_sifive.c [hunk 17]
```diff
@@ -119,8 +121,9 @@ static int i2c_sifive_write_msg(struct device *dev,
 
 	for (u32_t i = 0; i < msg->len; i++) {
 		/* Wait for a previous transfer */
-		while (i2c_sifive_busy(dev))
+		while (i2c_sifive_busy(dev)) {
 			;
```
Comment (carlescufi) [inline drivers/i2c/i2c_sifive.c]: I don't think you need the `;` anymore

File: drivers/i2c/i2c_sifive.c [hunk 18]
```diff
@@ -140,8 +143,9 @@ static int i2c_sifive_write_msg(struct device *dev,
 		sys_write8(command, I2C_REG(config, REG_COMMAND));
 
 		/* Wait for a previous transfer */
-		while (i2c_sifive_busy(dev))
+		while (i2c_sifive_busy(dev)) {
 			;
```
Comment (carlescufi) [inline drivers/i2c/i2c_sifive.c]: I don't think you need the `;` anymore

File: drivers/i2c/i2c_sifive.c [hunk 19]
```diff
@@ -161,8 +165,9 @@ static int i2c_sifive_read_msg(struct device *dev,
 
 	i2c_sifive_send_addr(dev, addr, SF_TX_READ);
 
-	while (i2c_sifive_busy(dev))
+	while (i2c_sifive_busy(dev)) {
 		;
```
Comment (carlescufi) [inline drivers/i2c/i2c_sifive.c]: I don't think you need the `;` anymore

File: drivers/i2c/i2c_sifive.c [hunk 20]
```diff
@@ -183,8 +188,9 @@ static int i2c_sifive_read_msg(struct device *dev,
 		sys_write8(command, I2C_REG(config, REG_COMMAND));
 
 		/* Wait for the read to complete */
-		while (i2c_sifive_busy(dev))
+		while (i2c_sifive_busy(dev)) {
 			;
```
Comment (carlescufi) [inline drivers/i2c/i2c_sifive.c]: I don't think you need the `;` anymore

File: drivers/rtc/rtc_ll_stm32.c [hunk 21]
```diff
@@ -238,8 +238,9 @@ static int rtc_stm32_init(struct device *dev)
 	LL_RCC_LSE_Enable();
 
 	/* Wait untill LSE is ready */
-	while (LL_RCC_LSE_IsReady() != 1)
+	while (LL_RCC_LSE_IsReady() != 1) {
 		;
```
Comment (carlescufi) [inline drivers/rtc/rtc_ll_stm32.c]: I don't think you need the `;` anymore

File: drivers/serial/uart_miv.c [hunk 22]
```diff
@@ -155,8 +155,9 @@ static void uart_miv_poll_out(struct device *dev,
 {
 	volatile struct uart_miv_regs_t *uart = DEV_UART(dev);
 
-	while (!(uart->status & STATUS_TXRDY_MASK))
+	while (!(uart->status & STATUS_TXRDY_MASK)) {
 		;
```
Comment (carlescufi) [inline drivers/serial/uart_miv.c]: I don't think you need the `;` anymore

File: drivers/serial/uart_ns16550.c [hunk 23]
```diff
@@ -430,8 +432,9 @@ static void uart_ns16550_poll_out(struct device *dev,
 					   unsigned char c)
 {
 	/* wait for transmitter to ready to accept a character */
-	while ((INBYTE(LSR(dev)) & LSR_THRE) == 0)
+	while ((INBYTE(LSR(dev)) & LSR_THRE) == 0) {
 		;
```
Comment (carlescufi) [inline drivers/serial/uart_ns16550.c]: I don't think you need the `;` anymore

File: drivers/serial/uart_nsim.c [hunk 24]
```diff
@@ -90,8 +90,9 @@ static void uart_nsim_poll_out(struct device *dev, unsigned char c)
 	u32_t regs = DEV_CFG(dev)->regs;
 	/* wait for transmitter to ready to accept a character */
 
-	while (!(UART_GET_STATUS(regs) & TXEMPTY))
+	while (!(UART_GET_STATUS(regs) & TXEMPTY)) {
 		;
```
Comment (carlescufi) [inline drivers/serial/uart_nsim.c]: I don't think you need the `;` anymore

File: drivers/serial/uart_sifive.c [hunk 25]
```diff
@@ -84,8 +84,9 @@ static void uart_sifive_poll_out(struct device *dev,
 	volatile struct uart_sifive_regs_t *uart = DEV_UART(dev);
 
 	/* Wait while TX FIFO is full */
-	while (uart->tx & TXDATA_FULL)
+	while (uart->tx & TXDATA_FULL) {
 		;
```
Comment (carlescufi) [inline drivers/serial/uart_sifive.c]: I don't think you need the `;` anymore

File: drivers/serial/uart_stellaris.c [hunk 26]
```diff
@@ -217,8 +217,9 @@ static inline void disable(struct device *dev)
 	uart->ctl &= ~UARTCTL_UARTEN;
 
 	/* ensure transmissions are complete */
-	while (uart->fr & UARTFR_BUSY)
+	while (uart->fr & UARTFR_BUSY) {
 		;
```
Comment (carlescufi) [inline drivers/serial/uart_stellaris.c]: I don't think you need the `;` anymore

File: drivers/serial/uart_stellaris.c [hunk 27]
```diff
@@ -327,8 +329,9 @@ static void uart_stellaris_poll_out(struct device *dev,
 {
 	volatile struct _uart *uart = UART_STRUCT(dev);
 
-	while (!poll_tx_ready(dev))
+	while (!poll_tx_ready(dev)) {
 		;
```
Comment (carlescufi) [inline drivers/serial/uart_stellaris.c]: I don't think you need the `;` anymore

File: drivers/spi/spi_sifive.c [hunk 28]
```diff
@@ -93,8 +93,9 @@ int spi_config(struct device *dev, u32_t frequency, u16_t operation)
 
 void spi_sifive_send(struct device *dev, u16_t frame)
 {
-	while (SPI_REG(dev, REG_TXDATA) & SF_TXDATA_FULL)
+	while (SPI_REG(dev, REG_TXDATA) & SF_TXDATA_FULL) {
 		;
```
Comment (carlescufi) [inline drivers/spi/spi_sifive.c]: I don't think you need the `;` anymore

File: drivers/spi/spi_sifive.c [hunk 29]
```diff
@@ -103,8 +104,9 @@ u16_t spi_sifive_recv(struct device *dev)
 {
 	u32_t val;
 
-	while ((val = sys_read32(SPI_REG(dev, REG_RXDATA))) & SF_RXDATA_EMPTY)
+	while ((val = sys_read32(SPI_REG(dev, REG_RXDATA))) & SF_RXDATA_EMPTY) {
 		;
```
Comment (carlescufi) [inline drivers/spi/spi_sifive.c]: I don't think you need the `;` anymore

File: tests/drivers/uart/uart_basic_api/src/test_uart_fifo.c [hunk 30]
```diff
@@ -102,8 +102,10 @@ static int test_fifo_read(void)
 	TC_PRINT("Please send characters to serial console\n");
 
 	data_received = false;
-	while (data_received == false)
+	while (data_received == false) {
 		;
```
Comment (carlescufi) [inline tests/drivers/uart/uart_basic_api/src/test_uart_fifo.c]: I don't think you need the `;` anymore

File: tests/drivers/uart/uart_basic_api/src/test_uart_poll.c [hunk 31]
```diff
@@ -22,8 +22,9 @@ static int test_poll_in(void)
 
 	/* Verify uart_poll_in() */
 	while (1) {
-		while (uart_poll_in(uart_dev, &recv_char) < 0)
+		while (uart_poll_in(uart_dev, &recv_char) < 0) {
 			;
```
Comment (carlescufi) [inline tests/drivers/uart/uart_basic_api/src/test_uart_poll.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/timeout_order.c [hunk 32]
```diff
@@ -65,12 +65,13 @@ void test_timeout_order(void)
 	u32_t uptime = k_uptime_get_32();
 
 	/* sync on tick */
-	while (uptime == k_uptime_get_32())
+	while (uptime == k_uptime_get_32()) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/timeout_order.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/clock.c [hunk 33]
```diff
@@ -101,23 +105,25 @@ void test_clock_cycle(void)
...
 	/**TESTPOINT: cycle/uptime cross check*/
 	c0 = k_cycle_get_32();
 	ALIGN_MS_BOUNDARY;
 	t32 = k_uptime_get_32();
-	while (t32 == k_uptime_get_32())
+	while (t32 == k_uptime_get_32()) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/clock.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/clock.c [hunk 34]
```diff
@@ -101,23 +105,25 @@ void test_clock_cycle(void)
 	c32 = k_cycle_get_32();
 	/*break if cycle counter wrap around*/
 	while (k_cycle_get_32() > c32 &&
-	       k_cycle_get_32() < (c32 + sys_clock_hw_cycles_per_tick()))
+	       k_cycle_get_32() < (c32 + sys_clock_hw_cycles_per_tick())) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/clock.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/clock.c [hunk 35]
```diff
@@ -64,22 +66,24 @@ void test_clock_uptime(void)
...
 
 	/**TESTPOINT: uptime delta lower 32-bit*/
 	k_uptime_delta_32(&d64);
 	/* Note: this will stall if the systick period < 5ms */
-	while (k_uptime_delta_32(&d64) < 5)
+	while (k_uptime_delta_32(&d64) < 5) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/clock.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/clock.c [hunk 36]
```diff
@@ -64,22 +66,24 @@ void test_clock_uptime(void)
 	/**TESTPOINT: uptime delta*/
 	d64 = k_uptime_delta(&d64);
 	/* Note: this will stall if the systick period < 5ms */
-	while (k_uptime_delta(&d64) < 5)
+	while (k_uptime_delta(&d64) < 5) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/clock.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/clock.c [hunk 37]
```diff
@@ -40,21 +40,23 @@ void test_clock_uptime(void)
...
+	}
 
 	/**TESTPOINT: uptime elapse lower 32-bit*/
 	t32 = k_uptime_get_32();
-	while (k_uptime_get_32() < (t32 + 5))
+	while (k_uptime_get_32() < (t32 + 5)) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/clock.c]: I don't think you need the `;` anymore

File: tests/kernel/common/src/clock.c [hunk 38]
```diff
@@ -40,21 +40,23 @@ void test_clock_uptime(void)
 
 	/**TESTPOINT: uptime elapse*/
 	t64 = k_uptime_get();
-	while (k_uptime_get() < (t64 + 5))
+	while (k_uptime_get() < (t64 + 5)) {
 #if defined(CONFIG_ARCH_POSIX)
 		k_busy_wait(50);
 #else
 		;
```
Comment (carlescufi) [inline tests/kernel/common/src/clock.c]: I don't think you need the `;` anymore

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: tests/kernel/common/src/clock.c (hunk 38 above)
Comment (nashif) [reply tests/kernel/common/src/clock.c]: yeah, was aware of that, but was too lazy, busted :(

**Review by alexanderwachter — APPROVED** [review]

**Review by dcpleung — APPROVED** [review]

**Review by sathishkuttan — APPROVED** [review]
drivers/audio/intel_dmic LGTM

**Review by ydamigos — APPROVED** [review]

**Review by joerchan — APPROVED** [review]
Bluetooth part looks good.

**Review by erwango — APPROVED** [review]
\*stm32\*

**Review by carlescufi — APPROVED** [review]

**Pull request merged**